app/views/admin/organizations/edit.html.haml

```diff
@@ -1,45 +1,42 @@
-%h1.padded
+%h1.mt-4
   Edit
   = link_to @organization.name, admin_organization_url(@organization)
 
-.form.form-horizontal
-  = form_for [:admin, @organization] do |f|
-    = render partial: 'form', locals: { f: f }
 
-    .nested-form
+= form_for [:admin, @organization] do |f|
+  = render partial: 'form', locals: { f: f }
+  .row.mt-4
+    .col-md-6.mb-4
+      %fieldset
+        #admin-locations-fields
+          = f.fields_for :locations do |builder|
+            = render '/admin/organizations/location_fields', f: builder
+          = revised_link_to_add_fields "Add a location", f, :locations, class_name: "btn btn-primary"
+
+    .col-md-4
+      %a.btn.btn-primary{"aria-controls" => "new_bike_notification", "aria-expanded" => "false", "data-toggle" => "collapse", :href => "#new_bike_notification"}
+        New bike notification
+      #new_bike_notification.collapse
+        .form-group
+        = f.label :new_bike_notification
+        = f.text_area :new_bike_notification, class: "form-control"
+    .col-md-2
       %p
         %strong
           Locations
-      %p#us-country-code.other-value
+      %p#us-country-code
```

Comment:
This was hidden on the old view, probably should be hidden again.

Reply:
It doesn't need to hide and appear - it actually never needs to be seen, it only needs to be accessed by the js. It should, in all correctly be made a `data-` property on one of the fields rather than text - but regardless - it definitely doesn't need to be seen by the user.

Reply:
What the attribute should actually be doing is making it so that new address blocks start, by default, with US as the country. Which isn't happening right now - but should